event/main_listener.php

```diff
@@ -60,14 +60,16 @@ public static function getSubscribedEvents()
 	 * @param \phpbb\config\db_text    $config_text
 	 * @param \phpbb\language\language $language
 	 * @param \phpbb\template\template $template
+	 * @param \phpbb\mediaembed\collection\customsitescollection $customSites
 	 */
-	public function __construct(\phpbb\auth\auth $auth, \phpbb\config\config $config, \phpbb\config\db_text $config_text, \phpbb\language\language $language, \phpbb\template\template $template)
+	public function __construct(\phpbb\auth\auth $auth, \phpbb\config\config $config, \phpbb\config\db_text $config_text, \phpbb\language\language $language, \phpbb\template\template $template, \phpbb\mediaembed\collection\customsitescollection $customSites)
 	{
 		$this->auth = $auth;
 		$this->config = $config;
 		$this->language = $language;
 		$this->template = $template;
 		$this->config_text = $config_text;
+		$this->customSites = $customSites;
```

Comment:
we should avoid camelCase on this one for coding guidelines reason...so `custom_sites`

Also this field needs a property declaration above the constructor.